keyboards/3.0/pcm.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE keyboard SYSTEM '../dtd/ldmlKeyboard.dtd'>
+<keyboard locale="pcm" conformsTo="techpreview">
+	<version number="1.0.0" />
+	<names>
+		<name value="Nigerian Pidgin" />
+		<name value="Naijíriá Píjin" />
+	</names>
+	<settings fallback="omit" />
+	<keys>
+		<import base="cldr" path="techpreview/keys-Zyyy-punctuation.xml" />
+		<import base="cldr" path="techpreview/keys-Zyyy-currency.xml" />
+		<key id="grave" to="̀" /> <!-- TODO: move to NCR -->
+		<key id="acute" to="́" /> <!-- TODO: move to NCR -->
+
+		<!-- accented vowels -->
+		<key id="odot" to="ọ" />
+		<key id="Odot" to="Ọ" />
+		<key id="edot" to="ẹ" />
+		<key id="Edot" to="Ẹ" />
+
+		<!-- currency -->
+		<key id="naira" to="₦" />
+	</keys>
+
+	<layers form="iso">
+		<layer modifier="none">
+			<row
+				keys="grave 1 2 3 4 5 6 7 8 9 0 hyphen equal" />
+			<row keys="acute w e r t y u i o p open-square close-square" />
+			<row keys="a s d f g h j k l odot edot slash" />
+			<row keys="slash z c v b n m comma period semi-colon apos" />
+			<row keys="space" />
+		</layer>
+
+		<layer modifier="shift">
+			<row
+				keys="grave bang at hash dollar naira percent amp asterisk open-paren close-paren underscore plus" />
+			<row keys="acute W E R T Y U I O P open-curly close-curly" />
+			<row keys="A S D F G H J K L Odot Edot" />
+			<row keys="question Z C V B N M open-angle close-angle colon double-quote" />
+			<row keys="space" />
+		</layer>
```

Comment:
Shouldn't there also be a caps layer? I don't see in the spec preview that that would happen automatically.

Reply:
Yes, there should, and there is one in the original file

Reply:
Good catch

Reply:
Maybe we should say something like unless there is an explicit caps layer, caps case-maps everything to uppercase

Reply:
It could be a tool feature too though, generate a caps layer based on unicode casing.

Reply:
Leave it to tooling I think -- auto caps layer adds complexity to implementations and necessitates a bunch of orthography assumptions which would be problematic and could lead to frustrating implementation inconsistencies and bugs. Turkish i/ı/İ/I is just the beginning.

Reply:
Well the layout does specify a locale that could be used for the case mapping, although we also support noting down alternative locales. The conflicting goal here is to have minimal files. I am perfectly happy with leaving this to tooling rather than the implementation.

Reply:
> locale used for case mapping

That's very much locale data, unicode version specific.  yes, that does make the files a little less minimal, but once the file is there it's completely independent of all Unicode versions and all locale data situations, which is best for less user surprise. As probably a bad example but Cherokee and Georgian didn't originally have bicamerality and now they do, so caps behavior could change dramatically.